src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,191 @@
+package duke.tasklist;
+
+import java.util.ArrayList;
+
+import duke.DukeException;
+import duke.command.Command;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+/**
+ * Class for operations to task list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Default constructor for TaskList. Used in case of unable to locate file.
+     */
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    /**
+     * Constructor for TaskList.
+     *
+     * @param tasks ArrayList of tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Carry out operations according to commands from user.
```

Comment:
Command pattern mandates an execute method to be overriden by the subclasses (implementation). User would just need to use `command.execute()` to execute the command instead of making use of task list to do it (It is not task list's job to execute the task. It merely stores the tasks)